templates/documents/general_upload.html

```diff
@@ -33,6 +37,14 @@
 <input type="hidden" name="category_id" value="{$category_id}" />
 <input type="hidden" name="process" value="{$PROCESS}" />
 </form>
+<!-- Drag and drop uploader -->
```

Comment:
@robertdown That concerns me also since the repository is a copy of what I have running.  Would you like me to create a new branch? That way all of this is in a single commit? That is the only thing I can think of to help.